File: Exercise/Hard/09. Inconsistent Stones in Boxes/Code/Inconsistent Stones in Boxes.py
def find_inconsistent_box():
    for i in range(1, 1001):
        stones = 0  
        for j in range(1, 1001):
            if i == j:
                stones += i  
        if stones != i:
            return i  

    return -1  

inconsistent_box = find_inconsistent_box()
print(inconsistent_box)

File: Exercise/Hard/09. Inconsistent Stones in Boxes/Code/Inconsistent Stones in Boxes.cpp
#include <iostream>

using namespace std;

int findInconsistentBox() {
    for (int i = 1; i <= 1000; ++i) {
        int stones = 0;  
        for (int j = 1; j <= 1000; ++j) {
            if (i == j) {
                stones += i;  
            }
        }
        if (stones != i) {
            return i;  
        }
    }
    return -1; 
}

int main() {
    int inconsistentBox = findInconsistentBox();
    cout << inconsistentBox << endl;
    return 0;
}
